src/frontends/pytorch/src/op_table.cpp

```diff
@@ -107,6 +107,7 @@ OP_CONVERTER(translate_linalg_matrix_norm);
 OP_CONVERTER(translate_linalg_vector_norm);
 OP_CONVERTER(translate_linear);
 OP_CONVERTER(translate_linspace);
+OP_CONVERTER(translate_mv);
```

Comment:
We aim to keep this list in alphabetical order, please relocate this line to an adequate spot